### PR TITLE
final footer et reviews burgermenu change width et add favicon externatic

### PR DESCRIPTION
**Pull request by SOA-P**

### CODE REVIEW

**Review by hswiergiel — DISMISSED** [review]

File: frontend/package.json [hunk 1]
```diff
@@ -16,7 +16,10 @@
     "react": "^18.0.0",
     "react-burger-menu": "^3.0.8",
     "react-dom": "^18.0.0",
+    "react-feather": "^2.0.10",
```
Comment (LoicBrassart) [inline frontend/package.json:19]: Feather is included in Icons, you can remove this line

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
A useless lib to remove from package.json

File: frontend/package.json (hunk 1 above)
Comment (SOA-P) [reply frontend/package.json:19]: ok :)

**Review by hswiergiel — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
SnE!

**Pull request merged**